app/web/src/components/CadPackage/CadPackage.tsx

```diff
@@ -46,19 +46,27 @@ const CadPackage = ({
   const cadPackageConfig = cadPackageConfigs[cadPackage]
 
   return (
-    <button
+    <ButtonOrDiv
       onClick={onClick}
       className={
-        `grid grid-flow-col-dense items-center gap-2 text-gray-100 bg-opacity-30 ${
-          onClick && ' hover:bg-opacity-80 '
-        } ${cadPackageConfig?.buttonClasses} ` + className
+        `grid grid-flow-col-dense items-center gap-2 text-gray-100 bg-opacity-30 
+         ${cadPackageConfig?.buttonClasses} ` + className
       }
     >
       <div
         className={`${cadPackageConfig?.dotClasses} ${dotClass} rounded-full`}
       />
       {cadPackageConfig?.label}
-    </button>
+    </ButtonOrDiv>
+  )
+}
+
+// Returns a proper button if an onClick handler is passed in, or a div
+// if the element is meant to be a simple badge
+function ButtonOrDiv({ onClick, className, children }) {
+  return (onClick
+    ? <button className={className + ' hover:bg-opacity-80'} onClick={ onClick }>{ children }</button>
+    : <div className={className}>{ children }</div>
```

Comment:
@Irev-Dev I thought about simply leaving the button and removing the `cursor: pointer` if no `onClick` is provided, but I think this is more semantic and will prevent some future "button cannot be child of button" warnings in future.

Reply:
Looks good, dynamic tag is also an option
https://stackoverflow.com/questions/33471880/dynamic-tag-name-in-jsx-and-react

Reply:
Oh nice this is more the stuff I was looking for! In this case I'm cool with the way I did it because I also added the class name, but I'll keep this in mind in future yeah?

Reply:
Yup, thats what I was going for, just an FYI if you weren't aware of it 😁